### ISSUE TITLE
rrt_init fails ungracefully if scripts use packages not in CRAN

### ISSUE THREAD

**Issue by revodavid** [RevolutionAnalytics/checkpoint#54]
Trying to use rrt_init on a repo:

```
> rrt_init(interactive=FALSE)
Checking to see if repository exists already...
Checing to make sure rrt directory exists inside your repository...
Creating rrt directory /Users/david/R/RRT-example2/rrt/lib/x86_64-apple-darwin13.1.0/3.1
Looking for packages used in your repository...
Error in FUN(c("ggplot2", "data.table", "knitr", "xts", "shiny", "RRT" : 
  No valid packages in pkg
```

The error message comes from `miniCRAN::pkgDep("RRT")`, because RRT is not a package on CRAN. I had a file in my repository with the line:

``` R
require(RRT)
```

which was detected by rrt_init() and which ultimately led to the `pkgDep` error message.
